modules/kafka/pom.xml

```diff
@@ -34,6 +35,12 @@
     <version>1.5.1.final-SNAPSHOT</version>
     <url>http://ignite.apache.org</url>
 
+    <properties>
+        <kafka.version>0.9.0.0</kafka.version>
```

Comment:
We store all the versions numbers in {ignite}/parent/pom.xml.

Please define new or override existing versions there.